### DISCUSSION TITLE
Going back from spectating deactivates the spectate state

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16093] [Q&A] [answered]
As per title, I don't think it should do that as it's annoying to turn it on if you want to spectate multiple matches.

https://user-images.githubusercontent.com/53081750/146144178-e5e65d15-eff8-4e6b-9e8f-fdc60d0ab150.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16096
